src/AppBundle/EventListener/FixDefaultSchemaListener.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace AppBundle\EventListener;
+
+use Doctrine\Common\EventSubscriber;
+use Doctrine\ORM\Tools\Event\GenerateSchemaEventArgs;
+
+class FixDefaultSchemaListener implements EventSubscriber
+{
+    public function getSubscribedEvents()
+    {
+        return array(
```

Comment:
For consistency can we use `[]` instead of `array` please.

Reply:
Done 👍